ethcontract/Cargo.toml

```
@@ -17,20 +17,23 @@ to Ethereum smart contracts.
 name = "ethcontract"
 
 [features]
-default = ["derive", "http-tls", "ws-tls-tokio", "derive-http"]
+aws-kms = ["aws-sdk-kms"]
+default = ["derive", "http-tls", "ws-tls-tokio", "derive-http", "aws-kms"]
```

I had this setup for testing, but I don't think it makes sense for this to be a default feature:
```suggestion
default = ["derive", "http-tls", "ws-tls-tokio", "derive-http"]
```